buildtest/cli/buildspec.py

```diff
@@ -1010,6 +1011,18 @@ def show_buildspecs(test_names, configuration):
         )
 
 
+def show_failed_buildspecs(configuration):
+    """This is the entry point for ``buildtest buildspec show-fail`` command which will print content of
+    buildspec on name of failed tests.
+
+    Args:
+        configuration (buildtest.config.SiteConfiguration): Instance of SiteConfiguration class
+    """
+    results = Report()
```

Comment:
please invoke as `Report(report_file=report_file)` assuming you define `report_file` as an argument for this method